src/util/metadata/header_metadata.rs

```diff
@@ -52,7 +52,7 @@ pub fn load_metadata(
     debug_assert!(optional_mask.is_none() || metadata_spec.num_of_bits >= 8,"optional_mask is only supported for 8X-bits in-header metadata. Problematic MetadataSpec: ({:?})", metadata_spec);
 
     // metadata smaller than 8-bits is special in that more than one metadata value may be included in one AtomicU8 operation, and extra shift and mask is required
-    let res: usize = if metadata_spec.num_of_bits < BITS_IN_BYTE {
+    let res: usize = if metadata_spec.num_of_bits < 8 {
```

Comment:
Right. This change looks strange. I made the change as in other methods with the same pattern, they use literals rather than the constants. I was trying to make it consistent. I will change this (and other methods) as well.